docs/core/deploying/deploy-with-cli.md

```diff
@@ -1,201 +1,154 @@
+```terminal
+  _   _      _ _         __        __         _     _ _
+ | | | | ___| | | ___    \ \      / /__  _ __| | __| | |
+ | |_| |/ _ \ | |/ _ \    \ \ /\ / / _ \| '__| |/ _` | |
+ |  _  |  __/ | | (_) |    \ V  V / (_) | |  | | (_| |_|
+ |_| |_|\___|_|_|\___( )    \_/\_/ \___/|_|  |_|\__,_(_)
+                     |/
+```
 
- [!code-xml[globalization-invariant-mode](~/samples/snippets/core/deploying/xml/invariant.csproj)]
+## Framework-dependent deployment
 
-1. Create a Debug build of your app.
+For the .NET Core SDK 2.x CLI, this publish mode is the default mode for the basic `dotnet publish` command.
 
-   From the command line, use the [dotnet build](../tools/dotnet-build.md) command.
+When you publish your app as a *framework-dependent deployment* (FDD), a `<PROJECT-NAME>.dll` file is created in the `<CURRENT-FOLDER>/bin/<BUILD-CONFIGURATION>/<TFM>/publish/` folder. To run your app, navigate to the output folder and call the `dotnet <PROJECT-NAME>.dll` command. 
 
-1. After you've debugged and tested the program, create the files to be deployed with your app for each platform that it targets.
+Your app is configured to target a specific version of .NET Core, and that runtime is required on the machine where you want to run your app. For example, if your app targets .NET Core 2.2, any machine that your app runs on must have the .NET Core 2.2 runtime installed. As stated in the [Publishing basics](#publishing-basics) section, you can edit your project file to change the default target framework or to target more than one framework.
 
-   Use the `dotnet publish` command for both target platforms as follows:
+## Framework-dependent executable
 
-      ```console
-      dotnet publish -c Release -r win10-x64
-      dotnet publish -c Release -r osx.10.11-x64
-      ```
+For the .NET Core SDK 3.x CLI, this publish mode is the default mode for the basic `dotnet publish` command. You don't need to specify any other parameters as long as want to target the current operating system.
 
-   This creates a Release (rather than a Debug) version of your app for each target platform. The resulting files are placed in a subdirectory named *publish* that's in a subdirectory of your project's *.\bin\Release\netcoreapp2.1\<runtime_identifier>* subdirectory. Note that each subdirectory contains the complete set of files (both your app files and all .NET Core files) needed to launch your app.
+By publishing a *framework-dependent executable* (FDE), a platform-specific executable is created. Similar to the FDD publish mode, your app targets the version of .NET Core that your project uses or that is specified on the command line. Your app is still created as a `.dll` file (like with FDD), but, a host executable is also generated that is specific to the target platform. You can run this executable instead of the `dotnet name.dll` command.
```

Comment:
These two paragraphs have all the facts, but could be more clear. I think the key path for the reader is:

1. Both FDDs and FDEs require that a runtime is installed on the target machine.
1. Both FDDs and FDEs require a native host app for that platform. For an FDD, the hostapp is `dotnet`. FDEs include their own host app.
1. Normal versioning rules for the runtime apply (link to .NET Versioning article)

Reply:
Bill's list is good, but I think our customers may need to see "EXE" near "hostapp" to know what the heck that is. 

(and autocomplete thinks it is hostage ;-)

Reply:
Some nits:
- FDE shouldn't be italicized.
- no comma after "but"
The order of the previous two paragraphs is reversed, and this could be tightened considerably, as Bill suggests.